src/types/PagoPaTypes.ts

```diff
@@ -0,0 +1,70 @@
+import * as t from "io-ts";
+import {
+  EmailString,
+  PatternString,
+  WithinRangeString
+} from "italia-ts-commons/lib/strings";
+import { Iban, Importo } from "./CommonTypes";
+
+export const Text16 = WithinRangeString(1, 16);
+export type Text16 = t.TypeOf<typeof Text16>;
+
+export const Text35 = WithinRangeString(1, 35);
+export type Text35 = t.TypeOf<typeof Text35>;
+
+export const Text70 = WithinRangeString(1, 70);
+export type Text70 = t.TypeOf<typeof Text70>;
+
+export const Text140 = WithinRangeString(1, 140);
+export type Text140 = t.TypeOf<typeof Text140>;
+
+export const ISODate = PatternString(
+  "[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}"
```

Comment:
use the `DateFromString` type, ref. https://github.com/teamdigitale/italia-ts-commons/blob/master/src/__tests__/dates.test.ts